maintainers/maintainer-list.nix

```diff
@@ -7266,12 +7272,6 @@
     githubId = 20658981;
     name = "Jarosław Wygoda";
   };
-  jyooru = {
-    email = "joel@joel.tokyo";
-    github = "jyooru";
```

Comment:
https://github.com/jyooru still exists. Can you provide evidence that you are the same person as @jyooru